Orchestration/Common/Helper.psm1

```diff
@@ -123,10 +126,17 @@ Function ConvertTo-AbsolutePath() {
     # Test if the path is valid
     $fileExists = Test-Path -Path $fullFilePath;
 
-    # Return path if file exists (truthy)
-    if($fileExists -eq $true) {
+    # Return path if file exists (truthy) and SkipFilePathExistenceCheck
```

Comment:
It is not truthy because you are checking for "true" condition, truthy accounts for null as well. Update the comment please

Reply:
Updated the comments.